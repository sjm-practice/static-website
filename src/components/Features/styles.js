const styles = () => ({
  container: {
    width: "100%",
    height: "700px",
    padding: "10px",
    background: "lightblue",
  },
});

export default styles;
