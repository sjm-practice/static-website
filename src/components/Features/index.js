import React from "react";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";
import styles from "./styles";

const Features = ({ classes }) => {
  return (
    <div id="Features" className={classes.container}>
      <Typography variant="h5">Features</Typography>
    </div>
  );
};

export default withStyles(styles)(Features);
