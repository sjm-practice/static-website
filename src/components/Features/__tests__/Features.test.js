/* eslint-env jest */
import React from "react";
import TestRenderer from "react-test-renderer";
import Features from "../index";

describe("<Features />", () => {
  it("should render and match snapshot", () => {
    const testRenderer = TestRenderer.create(<Features />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
