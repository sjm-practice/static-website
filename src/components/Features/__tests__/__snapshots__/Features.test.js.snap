// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Features /> should render and match snapshot 1`] = `
<div
  className="Features-container-1"
  id="Features"
>
  <h5
    className="MuiTypography-root-2 MuiTypography-h5-18"
  >
    Features
  </h5>
</div>
`;
